ws/request.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/python3
+
+import requests
```

Comment:
Nice library for a standalone program, but I think in the tests you need to make assertions about sent requests by a client and received requests on a server side, so you'd need deproxy